app/auto-eligibility/register-your-interest/check-eligibility.js

```diff
@@ -0,0 +1,43 @@
+const customerDbTable = require('../db/customer.db.table')
+
+const checkEligibility = async (sbi, crn, businessEmail) => {
+  console.log(`Checking eligibility: ${JSON.stringify({ sbi, crn, businessEmail })}`)
+  const customers = await customerDbTable.findAllBySbiOrBusinessEmail(sbi, businessEmail)
+  if (customers
+    .filter(customer => customer.sbi === sbi)
+    .some(customer => typeof customer.waiting_updated_at !== 'undefined')
+  ) {
+    console.log(`Sbi already registered: ${JSON.stringify({ sbi })}`)
+    return {
+      sbi,
+      crn,
+      businessEmail,
+      sbiAlreadyRegistered: () => true,
+      isEligible: () => false,
+      businessEmailHasMultipleDistinctSbi: () => false,
+      alreadyOnWaitingList: () => false,
+      hasAccessGranted: () => false
+    }
+  }
+  const eligibleCustomer = customers
+    .filter(customer => customer.business_email === businessEmail)

```

Comment:
may need to be careful here on the lowercase bug we have as well

Reply:
for example if we change the sequalize query to ensure the where condition is in lowercase, will the resultset return the value back to uppercase ->

can be looked at as part of https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/145631